src/main/java/uk/gov/companieshouse/api/accounts/service/impl/TnepValidationServiceImpl.java

```diff
@@ -55,19 +55,10 @@ public boolean validate(String ixbrl, String location) {
             Results results = validatIxbrlAgainstTnep(ixbrl, location);
 
             if (hasPassedTnepValidation(results)) {
-                addToLog(false, null, results, location,
-                    "Ixbrl is valid. It has passed the TNEP validation");
-
                 isIxbrlValid = true;
-
-            } else {
-                addToLog(true, null, results, location,
-                    "Ixbrl is invalid. It has failed the TNEP validation");
             }
 
         } catch (Exception e) {
```

Comment:
While we removed the addToLog call, I think it would be a good idea to still include an error log in the case of an exception.